deps/oculus-mobile/src/oculus-context.cpp

```diff
@@ -335,9 +340,13 @@ NAN_METHOD(OculusMobileContext::WaitGetPoses) {
               } else if (remoteCaps.ControllerCapabilities & (ovrControllerCaps_RightHand|ovrControllerCaps_ModelOculusGo)) {
                 ovrTracking tracking;
                 vrapi_GetInputTrackingState(oculusMobileContext->ovrState, capsHeader.DeviceID, predictedDisplayTime, &tracking);
-                ovrVector3f pivot{0.2, 1, 0};
-                ovrVector3f point{0.2, 1, -0.2};
-                tracking.HeadPose.Pose.Position = ovrVector3f_RotateAboutPivot(&tracking.HeadPose.Pose.Orientation, &pivot, &point);
+                // 3DOF Controller Tracking
+                if (remoteCaps.ControllerCapabilities & ovrControllerCaps_ModelOculusGo ||
+                    remoteCaps.ControllerCapabilities & ovrControllerCaps_ModelGearVR) {
+                  ovrVector3f pivot{-0.2, 1, 0};
+                  ovrVector3f point{-0.2, 1, -0.2};
```

Comment:
This one is for the right hand so it should be pivoted `0.2, 1, -0.2` I think.

Reply:
fixed